dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,955 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- 
+	The contents of this file are subject to the license and copyright detailed 
+	in the LICENSE and NOTICE files at the root of the source tree and available 
+	online at http://www.dspace.org/license/ 
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:oaire="http://namespace.openaire.eu/schema/oaire/" xmlns:datacite="http://datacite.org/schema/kernel-4"
+    xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:doc="http://www.lyncode.com/xoai"
+    xmlns:rdf="http://www.w3.org/TR/rdf-concepts/" version="1.0">
+    <xsl:output omit-xml-declaration="yes" method="xml" indent="yes"/>
+    
+    <!--  Please note that this crosswalk is mostly a backport from the DSpace 7 version 
+    	  available here https://github.com/DSpace/DSpace/blob/master/dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl
+          adapted to work with the limitation of the flat data model of DSpace pre version 7 -->	
+	<xsl:template match="/">
+		<oaire:resource
+			xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning"
+			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+			xsi:schemaLocation="http://namespace.openaire.eu/schema/oaire/ https://www.openaire.eu/schema/repo-lit/4.0/openaire.xsd">
+
+			<!-- DC TITLE -->
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='title']/doc:element/doc:field[@name='value']">
+				<datacite:title>
+					<xsl:value-of select="." />
+				</datacite:title>
+			</xsl:for-each>
+			
+            <!-- datacite:creator -->
+            <xsl:apply-templates
+                select="doc:metadata/doc:element[@name='dc']/doc:element[@name='contributor']/doc:element[@name='author']"
+                mode="datacite"/>			
+
+			<!-- ISSUE DATE -->
+			<datacite:date>
+				<xsl:attribute name="dateType">Issued</xsl:attribute>
+				<xsl:value-of select="doc:metadata/doc:element[@name='dc']/doc:element[@name='date']/doc:element[@name='issued']/doc:element/doc:field[@name='value']/text()" />
+			</datacite:date>
+			
+			<!-- EMBARGO PERIOD DATES -->
+
+			<xsl:if test="contains(doc:metadata/doc:element[@name='others']/doc:field[@name='drm']/text(),'embargoed')">
+                <datacite:dates>
+                    <datacite:date>
+                    	<xsl:attribute name="Accepted"/>
+                        <xsl:value-of select="substring-after(doc:metadata/doc:element[@name='others']/doc:field[@name='drm']/text(),'|||')"/>
+                    </datacite:date>    
+                    <datacite:date>
+                    	<xsl:attribute name="Available"/>
+                        <xsl:value-of select="doc:metadata/doc:element[@name='dc']/doc:element[@name='date']/doc:element[@name='accessioned']/doc:element/doc:field[@name='value']/text()"/>
```

Comment:
It should be <xsl:attribute name="dateType">Available</xsl:attribute> and should contains the embargo end date from the drm field. Please see: https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_embargoenddate.html